packages/tools/examples/contourApi/index.ts

```diff
@@ -0,0 +1,530 @@
+const volumeLoaderScheme = 'cornerstoneStreamingImageVolume'; // Loader id which defines which volume loader to use
+const volumeId = `${volumeLoaderScheme}:${volumeName}`; // VolumeId with loader id + volume id
+const renderingEngineId = 'myRenderingEngine';
+
+const viewportIds = ['CT_STACK', 'CT_VOLUME_SAGITTAL'];
+
+// ======== Set up page ======== //
+setTitleAndDescription(
+  'Planar Freehand Annotation Tool',
+  'Here we demonstrate how to use the Planar Freehand Annotation Tool to draw 2D open and closed ROIs'
+);
+
+const size = '500px';
+const content = document.getElementById('content');
+const viewportGrid = document.createElement('div');
+
+viewportGrid.style.display = 'flex';
+viewportGrid.style.display = 'flex';
+viewportGrid.style.flexDirection = 'row';
+
+const element1 = document.createElement('div');
+const element2 = document.createElement('div');
+element1.style.width = size;
+element1.style.height = size;
+element2.style.width = size;
+element2.style.height = size;
+
+// Disable right click context menu so we can have right click tool
+element1.oncontextmenu = (e) => e.preventDefault();
+element2.oncontextmenu = (e) => e.preventDefault();
+
+viewportGrid.appendChild(element1);
+viewportGrid.appendChild(element2);
+
+content.appendChild(viewportGrid);
+
+createInfoSection(content, { title: 'Drawing' })
+  .addInstruction('Left click and drag to draw a contour')
+  .openNestedSection()
+  .addInstruction(
+    'If you join the contour together it will be closed, otherwise releasing the mouse will create an open contour (freehand line)'
+  );
+
+createInfoSection(content, { title: 'Editing' })
+  .addInstruction(
+    'Left click and drag on the line of an existing contour to edit it'
+  )
+  .openNestedSection()
+  .addInstruction('Closed Contours')
+  .openNestedSection()
+  .addInstruction(
+    'Drag the line and a preview of the edit will be displayed. Release the mouse to complete the edit. You can cross the original contour multiple times in one drag to do a complicated edit in one movement.'
+  )
+  .closeNestedSection();
+
+createInfoSection(content, { title: 'Contour Utilities' })
+  .addInstruction(
+    'Find Contour Holes: Analyzes contours to detect holes within them. Results are logged to console.'
+  )
+  .addInstruction(
+    'Supersample Polylines: Adds interpolated points between existing points to achieve smoother contours with target spacing of 1.0 units.'
+  )
+  .addInstruction(
+    'Remove Small Islands: Filters out closed contours smaller than area threshold (100 square units). Only affects closed contours, preserves open polylines.'
+  )
+  .addInstruction(
+    'Decimate Polylines: Simplifies polylines by removing points using Ramer-Douglas-Peucker algorithm with epsilon tolerance of 2.0 units. Shows reduction percentage.'
+  );
+
+addButtonToToolbar({
+  title: 'Find Contour Holes',
+  onClick: () => {
+    const annotations = cornerstoneTools.annotation.state.getAllAnnotations();
```

Comment:
all these apis are for segmentations, so it should work like 

`cornerstoneTools.utilities.contours.removeHoles({ segmentationId, segmentIndex})` (which behind the scene uses your find and remove the other thing)